extensions/wikia/UserLogin/css/UserLoginDropdown.globalNavigation.scss

```diff
@@ -54,7 +54,7 @@
 	}
 
 	.WikiaForm {
-		padding: 0 $large-content-padding;
+		padding: 0 $large-content-padding $large-content-padding $large-content-padding;
```

Comment:
Our guidelines specify it should be: 

`padding: 0 $large-content-padding $large-content-padding`

The 3rd `$large-content-padding` is redundant b/c css will take the already specified "right" value and apply it to the "left" value.